### PR TITLE
update scipy to 0.18.1

### PR DESCRIPTION
**Pull request by zym1010**
for master branch of #18

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ source:
   md5: d70e7f533622ab705bc016dac328d93e
 
 build:
-  number: 202
+  number: 0
```
Comment (jakirkham) [inline recipe/meta.yaml]: Needs to be `200`.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -13,7 +13,7 @@ source:
   md5: d70e7f533622ab705bc016dac328d93e
```
Comment (jakirkham) [inline recipe/meta.yaml]: Please update the checksum.

Comment (jakirkham) [inline recipe/meta.yaml]: Also please switch to sha256 from [pypi.io]( https://pypi.io/ ).

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipe/meta.yaml]: xref: https://conda-forge.hackpad.com/BLAS-Numpy-Friends-86De62hoNdT

Comment (zym1010) [inline recipe/meta.yaml]: done. thanks for the info about versioning.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -10,10 +10,10 @@ package:
 source:
   fn: scipy-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/s/scipy/scipy-{{ version }}.tar.gz
-  md5: d70e7f533622ab705bc016dac328d93e
+  md5: 5fb5fb7ccb113ab3a039702b6c2f3327
```
Comment (jakirkham) [inline recipe/meta.yaml]: Please switch to sha256.

xref: https://pypi.org/project/scipy/0.18.1/#files

File: recipe/meta.yaml (hunk 2 above)
Comment (zym1010) [inline recipe/meta.yaml]: don't understand. you mean change 'md5: xxxxxx' to 'sha256: xxxxxxxxx'?

**Comment by jakirkham** [comment]
One last change and this should be good. Please add another PR against the branch `v0.18_np1.10`.

File: recipe/meta.yaml (hunk 3 above)
Comment (zym1010) [inline recipe/meta.yaml]: how? change the numbers after `md5` to be the sha256 checksum?

File: recipe/meta.yaml [hunk 4]
```diff
@@ -10,10 +10,10 @@ package:
 source:
   fn: scipy-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/s/scipy/scipy-{{ version }}.tar.gz
-  md5: d70e7f533622ab705bc016dac328d93e
+  md5: 8ab6e9c808bf2fb3e8576cd8cf07226d9cdc18b012c06d9708429a821ac6634e
```
Comment (jakirkham) [inline recipe/meta.yaml]: The key needs to change too. `md5` -> `sha256`

File: recipe/meta.yaml (hunk 2 above)
Comment (jakirkham) [inline recipe/meta.yaml]: Correct.

File: recipe/meta.yaml (hunk 4 above)
Comment (zym1010) [inline recipe/meta.yaml]: thanks. done.

File: recipe/meta.yaml (hunk 3 above)
Comment (jakirkham) [inline recipe/meta.yaml]: No you have to change the key too. What you said above was correct.

**Comment by jakirkham** [comment]
Thanks @zym1010. LGTM.

File: recipe/meta.yaml (hunk 2 above)
Comment (jakirkham) [inline recipe/meta.yaml]: Of course. Sorry this one is a bit odd.

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipe/meta.yaml]: Of course. Sorry this one is a bit odd.

**Pull request merged**